<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMain</class>
 <widget class="QMainWindow" name="FrmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Food Delivery Organizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuItem">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAdd_Item"/>
    <addaction name="actionEdit_Item"/>
    <addaction name="actionRemove_item"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuItem"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Item"/>
   <addaction name="actionEdit_Item"/>
   <addaction name="actionRemove_item"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Item">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/add.png</normaloff>:/add.png</iconset>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </action>
  <action name="actionEdit_Item">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/edit.jpg</normaloff>:/edit.jpg</iconset>
   </property>
   <property name="text">
    <string>Edit Item</string>
   </property>
  </action>
  <action name="actionRemove_item">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/remove.png</normaloff>:/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove item</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>FrmMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
